aw_datastore/datastore.py

```diff
@@ -56,8 +57,10 @@ def __init__(self, datastore: Datastore, bucket_id: str) -> None:
     def metadata(self) -> dict:
         return self.ds.storage_strategy.get_metadata(self.bucket_id)
 
-    def get(self, limit: int = 10**4,
+    def get(self, limit: int = -1,
             starttime: datetime=None, endtime: datetime=None) -> List[Event]:
+        if limit < 0:
+            limit = 10**4
```

Comment:
You didn't do this, but don't we use sys.maxsize on other datastores? Might be a better choice.